lib/Minz/Session.php

```diff
@@ -35,13 +36,26 @@ public static function param($p, $default = false) {
 	 * @param $v la valeur à attribuer, false pour supprimer
 	 */
 	public static function _param($p, $v = false) {
+		session_start();
 		if ($v === false) {
 			unset($_SESSION[$p]);
 		} else {
 			$_SESSION[$p] = $v;
 		}
+		session_write_close();
 	}
 
+	public static function _params($keyValues) {
```

Comment:
Optimisation function, to avoid starting/closing the session for several parameters to change i a row